lib/tasks/active_stash.rake

```diff
@@ -65,23 +65,19 @@ namespace :active_stash do
     info("")
   end
 
-  desc "Login to stash workspace"
+  desc <<~STR
+  Log into a CipherStash workspace.
+
+  If given a single arg, workspace, a profile will be created for that workspace.
+  This is useful for logging into a workspace for the first time.
+
+  If a workspace is not provided, this command will load an existing profile and
+  then log in using that profile. The profile name defaults to "default", but can
+  be overridden with the `CS_PROFILE_NAME` env var or the `defaultProfile` opt in
+  `{cs_config_path}/config.json`.
+  STR
```

Comment:
What is returned if they don't have an existing profile and don't provide a workspace id?

Reply:
That's a very good point. I tested this by nuking my local `.cipherstash` dir (after making a copy... we should add `CS_CONFIG_PATH` support to StashRB).

It prompts a log in and then gives an error that's not super informative:
```
CipherStash::Client::Error::AuthenticationFailure: access token scopes did not match what we requested (wanted ["collection.create", "collection.delete", "collection.info", "collection.list", "document.delete", "document.get", "document.put", "document.query", "offline_access", "ws:"], got ["collection.create", "collection.delete", "collection.info", "collection.list", "document.delete", "document.get", "document.put", "document.query", "offline_access"])
/Users/drew/cipherstash/activestash/lib/tasks/active_stash.rake:80:in `block (2 levels) in <main>'
Tasks: TOP => active_stash:login
```

That's a great catch @fimac. I'll take a look at making `CipherStash::Client.login` raise an error if it doesn't know which workspace to use.

Reply:
https://github.com/cipherstash/activestash/pull/87/commits/65cfbae4e36701ffb6db4d6aca213bbc4a25d0ed and https://github.com/cipherstash/ruby-client/pull/24 should handle this scenario more gracefully now @fimac.

Here's the error message that users will see if they attempt to run `rake active_stash:login` without the workspace arg before they have a profile to log in with:
```
Could not log in because a profile could not be loaded.

If you're logging into a workspace for the first time, make sure that you've provided a workspace
and a profile will be created for you.

Example: `rake active_stash:login[YourWorkspaceHere]`.

Otherwise, make sure that a profile named "default" exists or that a valid profile name has been
provided via the `CS_PROFILE_NAME` env var or the `defaultProfile` opt in `{cs_config_path}/config.json`.
```